src/components/Dashboard/constants.js

```diff
@@ -27,3 +29,16 @@ export const DESC_MAX_LENGTH = 140;
 export const TITLE_MAX_LENGTH = 31;
 export const TITLE_MAX_LENGTH_SMALL = 25;
 export const SUBHEADER_MAX_LENGTH_SMALL = 39;
+
+export const MAX_PIXELS = 3840;
+export const MAX_COLUMNS = 8;
+export const XL_NUM_COLUMNS = 6;
+export const XL_PIXELS = 1920;
```

Comment:
Was there a reason to define the pixels here instead of using MUI's breakpoints with sm/md/lg/etc?